client/state/selectors/is-site-eligible-for-customer-home.js

```diff
@@ -16,5 +14,6 @@ export default function isSiteEligibleForCustomerHome( state, siteId = null ) {
 	if ( ! siteId ) {
 		siteId = getSelectedSiteId( state );
 	}
-	return 'show' === abtest( 'customerHomePage' ) && canCurrentUserUseChecklistMenu( state, siteId );
+	// TODO: Refactor this so that it refers to Customer Home
+	return canCurrentUserUseChecklistMenu( state, siteId );
```

Comment:
Why not just rename `canCurrentUserUseChecklistMenu` (and the file `client/state/sites/selectors/can-current-user-use-checklist-menu.js`) and delete this file? This selector doesn't appear to be used anywhere else.

Reply:
I can't remember my logic for keeping them separate now. Unless we want to do something with the eligibility date, then I think you're right. I'll refactor it!